src/utils/backend/vaccine.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  BackendDataReturn,
+  BackendReturn,
+  DatabaseClient,
+} from "@utils/types/common";
+import { VaccineRecord } from "@utils/types/vaccine";
+
+export async function getVaccineRecordbyPersonId(
+  supabase: DatabaseClient,
+  personId: number
+): Promise<BackendDataReturn<VaccineRecord[]>> {
+  const { data, error } = await supabase
+    .from("vaccine_records")
+    .select("*")
+    .eq("person", personId)
+    .order("vaccination_date", { ascending: true });
+
+  if (error) {
+    // throw error;
+    // console.error(error);
```

Comment:
We usually log the errors too when they happen, so this line was actually correct.